draft-ietf-jsonpath-base.md

```diff
@@ -777,7 +772,8 @@ The following examples show the `wildcard` selector in use by a child segment.
 {: title="Wildcard selector examples"}
 
 The example above with the query `$.o[*, *]` shows that the wildcard selector may produce nodelists in distinct
```

Comment:
Well, the example does not show the parentheses.

s/but not/note that this is not the case/

Reply:
Sorry, I don't follow either of these comments. Anyway, I'm going to merge this PR, so let's take up the sequel elsewhere.